knotx-core/src/main/java/io/knotx/fragments/FragmentContentExtractor.java

```diff
@@ -15,21 +15,22 @@
  */
 package io.knotx.fragments;
 
-import io.knotx.dataobjects.Fragment;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Please apply formatting from the [coding conventions](https://github.com/Cognifide/knotx/blob/master/CONTRIBUTING.md#coding-conventions).
According to [google's java formatting](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing) rules:
```
Within each block the imported names appear in ASCII sort order.
```